SIRF_data_preparation/dataset_settings.py

```diff
@@ -10,19 +15,4 @@ class DatasetSettings:
 
 
 def get_settings(scanID: str):
```

Comment:
PS if participants want to have scanner-dependents subsets/params they could always check e.g. `if "Siemens_mMR" in str(data.path)`... should we do the same here?

Reply:
no. slice settings are going to depend on the phantom. And relying on our path seems like a dangerous strategy to me in any case.

Reply:
ignore my above comment, I hadn't see your separation between `DATA_SUBSETS` and `DATA_SLICES`.  Still, relying on pathnames seems bad to me. We could check .hs files etc, but then we're already solving the challenge. I wouldn't bother.

Reply:
technically this function is already solving the challenge (it defines scanner-dependent subsets).

I was just wondering if it should do it in a "better/more correct" way, i.e. the way we want participants to do it.

Reply:
"better" != "relying on file paths" 😄. Also, who knows what scanner names we will have in the final data (I don't). They should inspect the data instead (easiest is to look at number of views, but probably also TOF resolution).

Reply:
Excellent! gewd to know.